hare/algorithm_test.go

```diff
@@ -174,6 +175,44 @@ func (mev *mockEligibilityValidator) ValidateEligibilityGossip(context.Context,
 	return atomic.LoadInt32(&mev.valid) != 0
 }
 
+// We assert the the theshold X is met a long as there is one honest vote and
+// the sum of honest, dishonest and known equivocator is >= X.
+func TestCountInfoMeetsTrheshold(t *testing.T) {
```

Comment:
i think the test below is sufficient to deliver your point.
```suggestion
func TestCountInfoMeetsTrheshold(t *testing.T) {
        c := &CountInfo{}
	c.IncKnownEquivocator(1)
	require.False(t, c.Meet(3))
	c.IncDishonest(1)
	require.False(t, c.Meet(3))
	c.IncHonest(1)
	require.True(t, c.Meet(3))
}
```

Reply:
This test doesn't assert that at least one honest vote is required to meet the threshold.

I.E the following implementation of Meet would pass
```
ci.hCount+ci.dhCount+ci.keCount >= threshold
```